charts/consul/test/unit/server-statefulset.bats

```diff
@@ -1228,6 +1228,68 @@ load _helpers
   [ "${actual}" = "false" ]
 }
 
+#--------------------------------------------------------------------
+# global.acls.bootstrapToken
+
+@test "server/StatefulSet: when global.acls.bootstrapToken.secretKey!=null and global.acls.bootstrapToken.secretName=null, fail" {
+    cd `chart_dir`
+    run helm template \
+        -s templates/server-statefulset.yaml \
+        --set 'global.acls.bootstrapToken.secretName=' \
+        --set 'global.acls.bootstrapToken.secretKey=enterpriselicense' \
+        .
+  [ "$status" -eq 1 ]
+  [[ "$output" =~ "both global.acls.bootstrapToken.secretKey and global.acls.bootstrapToken.secretName must be set if one of them is provided." ]]
```

Comment:
I hadn't encountered the `=~` in BATS tests before. I assume it's some kind of approximately equal sign. However, it's a little difficult to Google for. Is my assumption correct?